modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/ChangeACSUrlTestCase.java

```diff
@@ -184,7 +179,7 @@ public void endTest() throws Exception {
 
         super.deleteSAML2WebSSOConfiguration(PORT_OFFSET_1, SECONDARY_IS_SAML_ISSUER_NAME);
         super.deleteServiceProvider(PORT_OFFSET_1, SECONDARY_IS_SERVICE_PROVIDER_NAME);
-        serverConfigurationManager.restoreToLastConfiguration(true);
+        serverConfigurationManager.restoreToLastConfiguration(false);
```

Comment:
is this intentional?

Reply:
No. Fixed with: https://github.com/wso2/product-is/pull/5158/commits/6fc7cd661abb956c87bf1c6b58b04e29aea5a7fc